frontend/src/components/core/MainMenu/MainMenu.tsx

```diff
@@ -223,9 +256,19 @@ function MainMenu(props: Props): ReactElement {
       ...S4AMenuOptions,
     ]
   } else {
+    menuOptions = [...menuOptions]
+
+    const showDeploy = props.isDeployable && isLocalhost()
+    if (showDeploy) {
```

Comment:
I won't be able to test if BaseWeb opens the menu because that's baseweb magic injecting the menu. The way to do it is to take the Popover's `content` prop, invoke it, and render the result, and check the `items` prop. I'll look into this.

Reply:
Oh, in my head it was just "modify props to set isDeployable to true, then check that the new snapshot has the deploy button in it". If it turns out to be more work then that then I'm okay to skip (so long as there aren't any regressions in the snapshot)

Reply:
Ah. The snapshot doesn't show the menu options. The only test is in Cypress. Ironically, the test is not failing, though I think it should...Going to figure out some more on a good solution.

Reply:
Oh wtf I guess I don't understand what MainMenu.test.tsx is even testing then (does it test anything?)

Reply:
It use to with Bootstrap cause the menu items were children instead of BaseWeb who has a content function that assembles the menu. I'll see if I can get some additional testing in there.

Reply:
Tests have checks for the labels, which is an improvement.